engine-types/src/lib.rs

```diff
@@ -25,8 +25,8 @@ mod v0 {
         vec::Vec,
     };
     pub use core::{
-        cmp::Ordering, fmt::Display, marker::PhantomData, mem, ops::Add, ops::Div, ops::Mul,
-        ops::Sub, ops::SubAssign,
+        cmp::Ordering, fmt::Display, marker::PhantomData, mem, ops::Add, ops::AddAssign, ops::Div,
+        ops::Mul, ops::Sub, ops::SubAssign,
```

Comment:
Seems a bit odd to me that this had changed when none of the logic below had. Just commenting on that.

Reply:
The diff looks more complicated than it is because of the formatting causing wrapping. All that happened here is `ops::AddAssign` was added to the imports and then it is used to implement `AddAssign` for the `EthGas` type.